src/sif/motorscootercost.cc

```diff
@@ -435,7 +454,7 @@ Cost MotorScooterCost::EdgeCost(const baldr::DirectedEdge* edge,
 
   float speed_penalty = (speed > top_speed_) ? (speed - top_speed_) * 0.05f : 0.0f;
   float factor = 1.0f + (density_factor_[edge->density()] - 0.85f) +
-                 (road_factor_ * kRoadClassFactor[static_cast<uint32_t>(edge->classification())]) +
+                 2* (road_factor_ * kRoadClassFactor[static_cast<uint32_t>(edge->classification())]) +
```

Comment:
Could you move this `2` into a named constant so readers can have some hint as to why we're doubling here?

Reply:
let me think of a good name. Can you do changes to the code as well?